frontend/src/components/editor.tsx

```diff
@@ -114,6 +116,22 @@ const Editor = ({ markdown }: EditorProps) => {
               h6: 'text-base font-bold',
             },
             quote: 'border-l-4 border-slate-500 pl-4',
+            list: {
```

Comment:
**curious:** Is there really a benefit to using the `Lexical` rich-text editor over [existing Markdown libraries](https://github.com/remarkjs/react-markdown)?

What if we want to add images?
What if we want to support code-blocks?
What about tables, checkboxes, strikethrough?

I'm concerned this library is gonna add unnecessary complexity by manually having you implementing those things, when we could "do it in 1 line" by adding a library designed for [Markdown](https://github.com/remarkjs/react-markdown).

`Lexical` library makes sense if we wanted to implement more complex logic than exists in current Markdown, but I don't think we want that. Let me know if I'm missing something though.

**Side-note:**
- On mobile, the `Lexical` editor is treated as a text-box, which brings up the keyboard as soon as you click on the text, and lets you edit things even if the editor is "hidden".
- I tried playing around with the editor by adding headings like you would in a markdown editor: `## Hi` and it didn't render. Although we won't be getting to editing until later, so it's not really a priority/block.

Reply:
The whole page isn't responsive except the editor atm. I can't replicate your mobile behavior. Also the heading thing works for me. Make sure you hit space after.

if we just wanted a simple markdown editor I could've just used a textbox and a parser. Standard nowadays is WYSIWYG editors which is nice for ui/ux.

Reply:
For mobile: using Chrome app on an Android. We can test more later.
I shared a video of the heading thing externally.

As for WYSIWYG editors, although nice, I personally think we should go for whatever is easier and works, since we have plenty of work to do. Just look at GitHub. They do a Write/Preview section. But use your best judgement.

Reply:
We can add a raw markdown editor feature if we feel that it provides a better user experience.